nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/adls/ADLSConstants.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.processors.adls;
+
+import org.apache.nifi.components.PropertyDescriptor;
+import org.apache.nifi.processor.Relationship;
+import org.apache.nifi.processor.util.StandardValidators;
+
+
+public class ADLSConstants {
+
+    public static final int CHUNK_SIZE_IN_BYTES = 4000000;
+
+    public static final PropertyDescriptor PATH_NAME = new PropertyDescriptor.Builder()
+            .name("Path")
```

Comment:
Could you use ``.name()`` and ``.displayName()`` ?
Name cannot be changed once we release the processor because it'd break backward compatibility on existing workflow. But we can change the display name as it's only used in logs/UI. We usually use a script-friendly naming for the name field (no whitespace, lower case, etc). For instance: adls-overwrite-policy.

Reply:
Used name and displayname properly wherever applicable.